nixos/doc/manual/release-notes/rl-2103.xml

```diff
@@ -357,6 +357,13 @@
      and rebooting.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     The <literal>services.dnscrypt-proxy2</literal> module now takes the upstream's example configuration and updates it with the user's settings.
```

Comment:
I don't find this formulation very clear. Maybe something along "the module changed the default values of all options to match those in upstream's example config". The important info is that default values changed, not really how this works under the hood.